apps/meteor/tests/e2e/omnichannel/omnichannel-chat-transfers.spec.ts

```diff
@@ -306,20 +305,19 @@ test.describe('OC - Chat transfers [Manager role]', () => {
 	test.beforeAll(async ({ api }) => {
 		const [departmentA] = departments.map(({ data }) => data);
 
-		conversations = await Promise.all([
-			createConversation(api, {
-				agentId: `user1`,
-				departmentId: departmentA._id,
-			}),
-			createConversation(api, {
-				agentId: `user1`,
-				departmentId: departmentA._id,
-			}),
-			createConversation(api, {
-				agentId: `user1`,
-				departmentId: departmentA._id,
-			}),
-		]);
+		const conversationA = await createConversation(api, {
+			agentId: `user1`,
```

Comment:
I dont know if I'm following or not, but its an issue related to race conditions, and now the conversations are being in sequence instead of parallel, what is the motivation behind these changes?